Makefile

```diff
@@ -111,7 +111,7 @@ coverage: testrunner
 
 test/lib/libtest.a:  $(EIGEN) $(TEST_OBJS)
 	$(Echo) "Making library"
-	$(Verb) ar -cvq $@ $(TEST_OBJS) > /dev/null
+	$(Verb) ar -cvr $@ $(TEST_OBJS) > /dev/null
```

Comment:
The Makefiles are a bit messy atm, as the QA team is in the midst of development/prototyping for some stuff. It would deserve some refactoring. I think, documenting this does not make sense now, as refactoring will change the structure of Makefiles a lot anyway. I can create a new task in clickup to refactor and document Makefiles. What do you think?

Btw, I think this libtest.a task should go away completely in the refactoring process.

Reply:
Sounds good. Yes, create a ClickUp (sub)task for it please.

Reply:
Task created under `General Improvements and Refactoring` https://app.clickup.com/t/2mnvcz